W-10169901 Published Grants-Management 1.19 installers (#4)
SalesforceFoundation/Grants-Management-Installers: README.md

> # Grants Management Installers

Reviewer: ```suggestion
# Grants Management Express Setup
```

> Welcome to the Grants Management installers!

Reviewer: ```suggestion
```

> Grants Management streamlines the grantmaking process and helps grantmakers cultivate and manage relationships with their grantees. Leveraging Salesforce Communities, Grants Management makes grantmaking a breeze through an intuitive, easy-to-use interface and backend automation.

Reviewer: ```suggestion
This read-only repository contains CumulusCI automation and unmanaged metadata components used in the Grants Management MetaDeploy installer.
```

> _IMPORTANT_: Salesforce does not recommend installing upgrades directly in production organizations. Instead, install Grants Management in one of your free sandbox environments or a developer organization first.

Reviewer: ```suggestion
```

> Once you've installed Grants Management, check out these resources:

Reviewer: ```suggestion
```

> -   [Grants Management Documentation](https://powerofus.force.com/s/article/GM-Documentation)

Reviewer: ```suggestion
```

> -   [Salesforce Partner Community - Grants Management Experts Group](https://partners.salesforce.com/_ui/core/chatter/groups/GroupProfilePage?g=0F94V000000LpYeSAK)

Reviewer: ```suggestion
Visit the [Grants Management customer group](https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000GvrgSAC) or [Salesforce Partner Community - Grants Management Experts Group](https://partners.salesforce.com/_ui/core/chatter/groups/GroupProfilePage?g=0F94V000000LpYeSAK) for any questions about this repository or Grants Management.
```

> Please visit the [Grants Management customer group](https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000GvrgSAC) on the Trailblazer Community for any questions about Grants Management.

Reviewer: ```suggestion
```

> ## Prerequisites for all installers

Reviewer: ```suggestion
```

> -   Grants Management Permission Set Licenses (PSLs) and Customer Community Plus Login licenses must be provisioned in your Salesforce org. For help, work with your sales representative.

Reviewer: ```suggestion
```

> -   [Install CumulusCI](https://cumulusci.readthedocs.io/en/latest/get_started.html) and [connect your org to CumulusCI](https://cumulusci.readthedocs.io/en/latest/connected_orgs.html).

Reviewer: ```suggestion
```

>     -   See the [CumulusCI Setup](https://trailhead.salesforce.com/en/content/learn/modules/cumulusci-setup) Trailhead module for step by step instructions how to [Review Base Requirements and Install Visual Studio Code](https://trailhead.salesforce.com/en/content/learn/modules/cumulusci-setup/review-base-requirements-install-visual-studio-code), [Install CumulusCI](https://trailhead.salesforce.com/en/content/learn/modules/cumulusci-setup/install-cumulusci), and [Connect to CumulusCI](https://trailhead.salesforce.com/en/content/learn/modules/cumulusci-setup/set-up-github-and-connect-to-cumulusci).

Reviewer: ```suggestion
```

> -   [Set Up My Domain](https://help.salesforce.com/s/articleView?id=sf.domain_name_setup.htm&type=5) in your Salesforce org.

Reviewer: ```suggestion
```

> -   [Enable Digital Experiences](https://help.salesforce.com/s/articleView?id=sf.workotcom_quemgmtenable_communities.htm&type=5) in your Salesforce org.

Reviewer: ```suggestion
```

> -   [Enable chatter](https://help.salesforce.com/s/articleView?id=sf.collab_enable.htm&type=5) for your Salesforce org.

Reviewer: ```suggestion
```

> > **NOTE**: Make sure to complete all [Prerequisites for all installers](#Prerequisites-for-all-installers) before attempting any installation.

Reviewer: ```suggestion
```

> # Installation errors

Reviewer: ```suggestion
```

> To get help with any installation error, immediately execute the `cci error info` command in the terminal and go to [help.salesforce.com](https://help.salesforce.com/), find the Support & Services area and log a technical support ticket with Salesforce. Include “Grants Management” in the subject line and include the output of `cci error info` in your comment.

Reviewer: ```suggestion
```

# Installers

> -   [Install Grants Management and Build Community](#Install-Grants-Management-and-Build-Community)

Reviewer: ```suggestion
-   [Grants Management and Build Community](#Install-Grants-Management-and-Build-Community)
```

Reviewer: @spelak-salesforce @sella24 Note: I also updated the section headers further down in the README. Will that affect the jump links in this list?

Author: @melissabarber  Probably.  Can you test the README after the changes are committed?

> -   [Install Grants Management](#Install-Grants-Management)

Reviewer: ```suggestion
-   [Grants Management](#Install-Grants-Management)
```

> -   [Install Grants Management Flows](#Install-Grants-Management-Flows)

Reviewer: ```suggestion
-   [Grants Management Flows](#Install-Grants-Management-Flows)
```

>     -   [Deploy Grants Management: Multi-Section Application Form (Sample) and subflow](#Deploy-Grants-Management-Multi-Section-Application-Form-Sample-and-subflow)

Reviewer: ```suggestion
    -   [Grants Management: Multi-Section Application Form (Sample) and subflow](#Deploy-Grants-Management-Multi-Section-Application-Form-Sample-and-subflow)
```

>     -   [Deploy Enhanced Application (Sample) Flow](#Deploy-Enhanced-Application-Sample-Flow)

Reviewer: ```suggestion
    -   [Enhanced Application (Sample) Flow](#Deploy-Enhanced-Application-Sample-Flow)
```

>     -   [Deploy Create Application Flow](#Deploy-Create-Application-Flow)

Reviewer: ```suggestion
    -   [Create Application Flow](#Deploy-Create-Application-Flow)
```

>     -   [Deploy Submit Application Flow](#Deploy-Submit-Application-Flow)

Reviewer: ```suggestion
    -   [Submit Application Flow](#Deploy-Submit-Application-Flow)
```

>     -   [Deploy Grants Management: Validate EIN (Sample) Flow](#Deploy-Grants-Management-Validate-EIN-Sample-Flow)

Reviewer: ```suggestion
    -   [Grants Management: Validate EIN (Sample) Flow](#Deploy-Grants-Management-Validate-EIN-Sample-Flow)
```

> ## Install Grants Management and Build Community

Reviewer: ```suggestion
## Grants Management and Build Community
```

| Deploy Grants Management: Multi-Section Application Form (Sample) Flows | Metadata | Required |
| Deploy Profiles                                                         | Metadata | Required |
| Deploy Submit Application Flow                                          | Metadata | Required |
| Deploy Grants Management: Validate EIN (Sample) Flow                    | Metadata | Required |
| Install Outbound Funds NPSP Extension                                   | Package  | Optional |
| Create the Funding Program Portal                                       | Other    | Required |
| Publish the Funding Program Portal                                      | Other    | Required |
| Install Funding Program Portal Navigation Menu                          | Metadata | Required |
| Add Grantseeker Profile to Funding Program Portal                       | Other    | Required |
| Grant Visibility to Community Users                                     | Metadata | Required |
| Add Funding Program Portal Branding                                     | Metadata | Required |
| Update Funding Program Portal Profile (Guest User) profile              | Metadata | Required |

> To install, open a terminal at this repository and execute the following command:

Reviewer: ```suggestion
```

```

> cci flow run customer_org_full --org <your connected Salesforce org>

Reviewer: ```suggestion
```

```

> ## Install Grants Management

Reviewer: ```suggestion
## Grants Management
```

> This will install Grants Management into your org.

Reviewer: ```suggestion
This installs Grants Management into your org.
```

Reviewer: @spelak-salesforce is the difference between this installer and the previous one, Install Grants Management and Build Community, that this one doesn't do anything with the community? Do customers chose to install this or the previous on? Should we call that out here?

| ----------------------------------------------------------------------- | -------- | -------- |
| Install SFDO Base 1.0                                                   | Package  | Required |
| Install Outbound Funds (Core)                                           | Package  | Required |
| Deploy Visual Assets                                                    | Metadata | Required |
| Install Grants Management                                               | Package  | Required |
| Deploy Grants Management: Enhanced Application (Sample) Flow            | Metadata | Required |
| Deploy Create Application Flow                                          | Metadata | Required |
| Deploy Grants Management: Multi-Section Application Form (Sample) Flows | Metadata | Required |
| Deploy Profiles                                                         | Metadata | Required |
| Deploy Submit Application Flow                                          | Metadata | Required |
| Deploy Grants Management: Validate EIN (Sample) Flow                    | Metadata | Required |
| Install Outbound Funds NPSP Extension                                   | Package  | Optional |

> To install, open a terminal at this repository and execute the following command:

Reviewer: ```suggestion
```

```

> cci flow run customer_org --org <your connected Salesforce org>

Reviewer: ```suggestion
```

```

> ## Install Grants Management Flows

Reviewer: ```suggestion
## Grants Management Flows
```

> Install or overwrite the selected unpackaged Grants Management flows in your org. After installation, you can access and activate earlier versions of previously-installed flows as needed. Be sure to create a backup of any changes before proceeding.

Reviewer: ```suggestion
```

> > **Additional Prerequisites**

Reviewer: ```suggestion
```

>

> > -   Grants Management version 1.17 is the minimum version required to (re)install any unpackaged Grants Management flows.

Reviewer: ```suggestion
```

> ### Deploy Grants Management: Multi-Section Application Form (Sample) and subflow

Reviewer: ```suggestion
### Grants Management: Multi-Section Application Form (Sample) and subflow
```

> To install or overwrite the Grants Management: Multi-Section Application Form (Sample) and subflow, open a terminal at this repository and execute the following command:

Reviewer: ```suggestion
The Grants Management: Multi-Section Application Form (Sample) flow shows a grantseeker an application with multiple sections and allows a grantseeker to save and return to the application. This flow is continuous until a grantseeker submits their application.
```

```

> cci task run deploy_multi_stage_app_flow --org <your connected Salesforce org>

Reviewer: ```suggestion
```

```

> ### Deploy Enhanced Application (Sample) Flow

Reviewer: ```suggestion
### Enhanced Application (Sample) Flow
```

> To install or overwrite the Deploy Enhanced Application (Sample) Flow, open a terminal at this repository and execute the following command:

Reviewer: ```suggestion
The Enhanced Application (Sample) flow condenses the default application experience using the Create Application flow from three screens to one screen. This is a sample flow and includes example application language, unmapped custom fields, and a file upload component within a single page.
```

```

> cci task run deploy_enhanced_application_flow --org <your connected Salesforce org>

Reviewer: ```suggestion
```

```

> ### Deploy Create Application Flow

Reviewer: ```suggestion
### Create Application Flow
```

> To install or overwrite the Create Application Flow, open a terminal at this repository and execute the following command:

Reviewer: ```suggestion
The Create Application flow automates how the application initially starts for your grantseeker. This involves retrieving information about the Funding Program and user’s Contact information, autofilling fields on the application, and creating the application record. Without this flow, a grantseeker can’t apply for a Funding Request without making permission updates to the Grantseeker Plus Login profile.
```

```

> cci task run deploy_funding_program_application_flow --org <your connected Salesforce org>

Reviewer: ```suggestion
```

```

> ### Deploy Submit Application Flow

Reviewer: ```suggestion
### Submit Application Flow
```

> To install or overwrite the Submit Application Flow, open a terminal at this repository and execute the following command:

Reviewer: ```suggestion
The Submit Application flow automates how a grantseeker’s application is submitted to an organization. The flow retrieves the application record, makes sure the user is both a site user and owner of the application, and updates the application before it is submitted. Without this flow, the grantseeker’s application can’t be submitted to your organization for review.
```

```

> cci task run deploy_submit_application_flow --org <your connected Salesforce org>

Reviewer: ```suggestion
```

```

> ### Deploy Grants Management: Validate EIN (Sample) Flow

Reviewer: ```suggestion
### Grants Management: Validate EIN (Sample) Flow
```

> To install or overwrite the Grants Management: Validate EIN (Sample) Flow, open a terminal at this repository and execute the following command:

Reviewer: ```suggestion
The Grants Management: Validate EIN (Sample) Flow allows your grants managers to search for and validate the EIN of an organization.```

Reviewer: @spelak-salesforce or @sella24 please check what I've written here for accuracy. The product documentation didn't have an explanation of this flow.

```

> cci task run deploy_validate_ein_flow --org <your connected Salesforce org>

Reviewer: ```suggestion
```

# Grants Management Express Setup

This read-only repository contains CumulusCI automation and unmanaged metadata components used in the Grants Management MetaDeploy installer.

Visit the [Grants Management customer group](https://trailhead.salesforce.com/trailblazer-community/groups/0F94S000000GvrgSAC) or [Salesforce Partner Community - Grants Management Experts Group](https://partners.salesforce.com/_ui/core/chatter/groups/GroupProfilePage?g=0F94V000000LpYeSAK) for any questions about this repository or Grants Management.

# Installers

> - [Grants Management and Build Community](#Grants-Management-and-Build-Community)

Reviewer: ```suggestion
- [Grants Management and Funding Program Portal Community](#Grants-Management-and-Funding-Program-Portal-Community)
```

> - [Grants Management](#Grants-Management)

Reviewer: ```suggestion
- [Grants Management, Only](#Grants-Management-Only)
```

- [Grants Management Flows](#Grants-Management-Flows)
  - [Grants Management: Multi-Section Application Form (Sample) and subflow](#Grants-Management-Multi-Section-Application-Form-Sample-and-subflow)
  - [Enhanced Application (Sample) Flow](#Enhanced-Application-Sample-Flow)
  - [Create Application Flow](#Create-Application-Flow)
  - [Submit Application Flow](#Submit-Application-Flow)
  - [Grants Management: Validate EIN (Sample) Flow](#Grants-Management-Validate-EIN-Sample-Flow)

> ## Grants Management and Build Community

Reviewer: ```suggestion
## Grants Management and Funding Program Portal Community
```

> This installs Grants Management into your org. The installer creates a Community with the Grantee Portal template and applies our recommended configurations.

Reviewer: ```suggestion
This installs the recommended setup of Grants Management into your org. In addition to installing Grants Management, this installer creates a Community with the Grantee Portal template and applies our recommended configurations.
```

| Deploy Grants Management: Multi-Section Application Form (Sample) Flows | Metadata | Required |
| Deploy Profiles                                                         | Metadata | Required |
| Deploy Submit Application Flow                                          | Metadata | Required |
| Deploy Grants Management: Validate EIN (Sample) Flow                    | Metadata | Required |
| Install Outbound Funds NPSP Extension                                   | Package  | Optional |
| Create the Funding Program Portal                                       | Other    | Required |
| Publish the Funding Program Portal                                      | Other    | Required |
| Install Funding Program Portal Navigation Menu                          | Metadata | Required |
| Add Grantseeker Profile to Funding Program Portal                       | Other    | Required |
| Grant Visibility to Community Users                                     | Metadata | Required |
| Add Funding Program Portal Branding                                     | Metadata | Required |
| Update Funding Program Portal Profile (Guest User) profile              | Metadata | Required |

> ## Grants Management

Reviewer: ```suggestion
## Grants Management, Only
```

> This installs Grants Management into your org.

Reviewer: ```suggestion
This installs Grants Management into your org and doesn't create the Funding Program Portal.
```
